packages/core/src/lib/media/media.service.ts

```diff
@@ -69,4 +69,13 @@ export class MediaService {
   isTouchScreen(): boolean {
     return 'ontouchstart' in document.documentElement ? true : false;
   }
+
+  isMobile(): boolean {
+    const media = this.getMedia();
+    if (media === 'mobile') {
```

Comment:
You could simply do:

`return media === 'mobile';`

I won't refude the PR for this though.

Reply:
done